packages/runtime/src/runtime/mock.tsx

```diff
@@ -0,0 +1,38 @@
+import type { EnsembleMockResponse } from "@ensembleui/react-framework";
+import { isObject, isNumber, isEmpty } from "lodash-es";
+
+export const mock = (evaluatedMockResponse: {
```

Comment:
Can we move this into the framework project alongside the rest of the mocking logic?